recipes/dimcli/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "dimcli" %}
+{% set version = "0.9.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dimcli-{{ version }}.tar.gz
+  sha256: a6c57ec1f2a1e64831f6339d69a36eb3b3cce72366a861371f639b5bc3e8549c
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - dimcli = dimcli.main_cli:main_cli
+    - dimcli_quicktest = dimcli.tests.quicktest:main
+  script:
+    {% set reqfile = "requirements.txt" %}
+    {% set reqsource = "dimcli.egg-info/requires.txt" %}
+    # IF {{ reqfile }} is NOT PRESENT, COPY {{ reqfile }} from {{ reqsource }}
+    - if [ ! -f {{ reqfile }} ] && [ -f {{ reqsource }} ]; then cp {{ reqsource }} {{ reqfile }}; fi  # [not win]
+    - copy dimcli.egg-info\requires.txt {{ reqfile }}  # [win]
+    - {{ PYTHON }} -m pip install . -vv
```

Comment:
Kudos for this bit of magic!

Reply:
Hehe! Not quite!

If you see carefully, I was not able to apply the "IF" conditionals for windows. Any help/suggestion on what I should have done?

Reply:
Windows has its own syntax that I usually find by looking through other recipes. IDK If you tried that. I don't see the problem with always copying requires though.